lib/chartmogul/customer.js

```diff
@@ -31,12 +31,12 @@ class Customer extends Resource {
 
   static notes (config, customerId, params, callback) {
     const path = util.expandPath(CustomerNote.path, []);
-    return Resource.request(config, 'GET', path, { ...params, customerId }, callback);
+    return Resource.request(config, 'GET', path, { ...params, customer_uuid: customerId }, callback);
   }
 
-  static createrNote (config, customerId, params, callback) {
+  static createNote (config, customerId, params, callback) {
     const path = util.expandPath(CustomerNote.path, []);
-    return Resource.request(config, 'POST', path, { ...params, customerId }, callback);
+    return Resource.request(config, 'POST', path, { ...params, customer_uuid: customerId }, callback);
```

Comment:
Perhaps it's a good idea to rename the variable for clarity
```suggestion
    return Resource.request(config, 'POST', path, { ...params, customer_uuid: customerUuid }, callback);
```